README.md

```diff
@@ -22,4 +22,4 @@ In this task, we use the [Bulma CSS framework](https://bulma.io/). It is already
 
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Open one more terminal and run tests with `npm test` to ensure your solution is correct.
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_goods-selector-js/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://davinchii.github.io/react_goods-selector-js/) and add it to the PR description.
```

Comment:
please fix link, I can't open your demo link.

Reply:
Use Demo link from PR description, i fixed it there after Max's comment